### PR TITLE
Fix properties in base inventory classes.

### PR DESCRIPTION
**Pull request by martynasslav**
Change property access modifier to private.
Add get methods.
Fix property initialization.

### CODE REVIEW

**Comment by 4rgetlahm** [comment]
I guess I have caused some confusion. As I understand Properties do not need any other separate getters or setters, as they have {get; set;}. You can check out a video on the Properties and their getters/setters (https://www.youtube.com/watch?v=EbW-1fPhXQE&ab_channel=MikeDane). Properties should be public.

**Review by 4rgetlahm — CHANGES_REQUESTED** [review]
I guess I have caused some confusion. As I understand Properties do not need any other separate getters or setters, as they have {get; set;}. You can check out a video on the Properties and their getters/setters (https://www.youtube.com/watch?v=EbW-1fPhXQE&ab_channel=MikeDane). Properties should be public.

File: object-group-game/Effect.cs [hunk 1]
```diff
@@ -4,14 +4,14 @@
 
 namespace object_group_game {
 	class Effect {
```
Comment (4rgetlahm) [inline object-group-game/Effect.cs:6]: C# please

File: object-group-game/Effect.cs [hunk 2]
```diff
@@ -4,14 +4,14 @@
...
-		public string Status { get; set; }
-		public string DisplayName { get; set; }
-		public int Value { get; set; }
+		public string Status { get; private set; }
+		public string DisplayName { get; private set; }
+		public int Value { get; private set; }
 
 		Effect(string status, string displayName, int value) {
```
Comment (4rgetlahm) [inline object-group-game/Effect.cs:11]: C# please

File: object-group-game/Inventory.cs [hunk 3]
```diff
@@ -4,6 +4,23 @@
 
 namespace object_group_game {
 	class Inventory {
-		public List<Item> Items { get; set; }
+		private List<Item> _items;
+		public List<Item> Items {
```
Comment (4rgetlahm) [inline object-group-game/Inventory.cs:8]: Do we really need cloning for this? (Just asking, I'm not sure about how you envision this)
(C# please)

**Review by 4rgetlahm — CHANGES_REQUESTED** [review]
Can you change from Java style to C#?

Also check comment about Inventory class

File: object-group-game/Inventory.cs (hunk 3 above)
Comment (martynasslav) [reply object-group-game/Inventory.cs:8]: I think list of items should not be manipulated outside the Inventory class. I believe returning a clone is the best solution for that. Now, when I think about it, auto-generated get method should be private and return the actual reference to the list for the class methods to be able to manipulate it. An additional method should be created to get it from outside the class. It sounds right in my mind, would appreciate a second opinion.

**Pull request merged**